site/homepage/content/release-notes.md

```diff
@@ -45,6 +45,10 @@ title = "Release Notes"
 * The *set command-handling protocol adapter instance* operation of the *Device Connection* API
   has been extended to support an additional parameter which can be used to indicate the
   maximum amount of time that the given information is to be considered valid.
+* The `hono-core` module no longer embeds the *OpenTracing API* classes by default.
```

Comment:
nor the Spring Crypto classes, for that matter

Reply:
True.